test/bats.bats

```diff
@@ -37,7 +37,7 @@ fixtures bats
   run bats "$FIXTURE_ROOT/passing.bats"
   [ $status -eq 0 ]
   [ ${lines[0]} = "1..1" ]
```

Comment:
All strings (unless you're absolutely sure it will be a single number or word) should be quoted, especially in conditional expressions.